docs/app/pages/components/components-sections/select/select-custom/snippets/app.html

```diff
@@ -0,0 +1,45 @@
+<div class="row">
+    <div class="col-md-5">
+
+        <ux-select-custom
+            [(selected)]="selected"
+            (filterChange)="filter = $event"
+            [allowNull]="allowNull"
+            [busy]="showBusyIndicator"
+            [width]="componentWidth"
+            [maxHeight]="componentMaxHeight"
+            [placeholder]="placeholder">
+
+            <ng-template #buttonContent>
+                <b>Selection:</b> {{ selected ? selected.name : '(none)' }}
+            </ng-template>
+
+            <ng-template #dropdownContent>
+
+                <div role="radiogroup"
+                        class="radio-button-container"
+                        tabindex="0"
+                        (keydown.space)="selectButton($event)"
+                        (keydown.up)="navigateUp($event)"
+                        (keydown.arrowUp)="navigateUp($event)"
+                        (keydown.down)="navigateDown($event)"
+                        (keydown.arrowDown)="navigateDown($event)">
+
+                    <ux-radio-button
+                        *ngFor="let option of optionList; let index = index"
+                        name="group"
+                        [value]="selected"
+                        (valueChange)="selected = $event"
+                        [option]="option"
+                        [hidden]="isHidden(option.name)"
+                        [class.focused]="focusIndex === index">
+                        {{ beforeHighlight(option.name) }}
+                        <span class="highlight">{{ highlightText(option.name)}}</span>
```

Comment:
I think the highlight color should be updated:

![image](https://user-images.githubusercontent.com/20795331/62701744-f708bf00-b9dc-11e9-8a86-51e33ea9adac.png)

The blue color is not in our colour palette, instead consider something like this (see [Facet Typehead List](http://localhost:8080/#/components/facets#facet-typeahead-list)):

![image](https://user-images.githubusercontent.com/20795331/62701800-143d8d80-b9dd-11e9-89d9-93c9b377b72e.png)

Reply:
The space around the highlight had already been fixed in my previous commit. I fixed the color.